extensions/wikia/Search/classes/IndexService/CrossWikiCore.php

```diff
@@ -246,6 +247,20 @@ protected function getIsPromotedWiki() {
 		];
 	}
 
+	/**
+	 * Get a flag that prevents a wiki from showing up in wiki search results.
+	 *
+	 * @see PLATFORM-3659
+	 * @return array
+	 */
+	protected function getIsHiddenWiki() {
+		global $wgExcludeWikiFromSearch;
+
+		return [
+			'hidden_wiki_b' => !empty( $wgExcludeWikiFromSearch )
```

Comment:
Please add this variable and some description to `includes/wikia/VariablesBase.php`

Reply:
@themech done in 73526fc9428